index.html

```diff
@@ -6,15 +6,24 @@
 	<meta name="viewport" content="width=device-width">
 	<title>SerSquirt's Website</title>
 	<link href="style.css" rel="stylesheet" type="text/css" />
+  <html lang="en-US">
```

Comment:
This is an extra `html` tag.  There is also one on line 2.  But the one on line 2 should have `lang="en-US"` like this one.

Reply:
Fixed!